Packs/Endace/TestPlaybooks/Endace-Test.yml

```diff
@@ -1,757 +1,776 @@
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 3400
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-  "18":
-    id: "18"
-    taskid: b87db823-6dec-40a9-8128-1bf759e7cdb7
-    type: title
-    task:
-      id: b87db823-6dec-40a9-8128-1bf759e7cdb7
-      version: -1
-      name: Test Done
-      type: title
-      iscommand: false
-      brand: ""
-    separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 3600
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-  "19":
-    id: "19"
-    taskid: 5e56d6fb-48ee-4760-86e7-6d756e1f4cc7
-    type: regular
-    task:
-      id: 5e56d6fb-48ee-4760-86e7-6d756e1f4cc7
-      version: -1
-      name: endace-download-pcap
-      description: Download a copy of the PCAP file from EndaceProbe if PCAP file
-        size is within the threshold value defined by filesizelimit.
-      script: Endace|||endace-download-pcap
-      type: regular
-      iscommand: true
-      brand: Endace
-    nexttasks:
-      '#none#':
-      - "15"
-    scriptarguments:
-      filename:
-        simple: ${Endace.Archive.Response.FileName}
-      filesizelimit: {}
-    separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 2820
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-  "20":
-    id: "20"
-    taskid: 70202d1e-2ff8-4768-83b4-2249e55fd85c
-    type: regular
-    task:
-      id: 70202d1e-2ff8-4768-83b4-2249e55fd85c
-      version: -1
-      name: endace-create-archive
-      description: Create an archive task to archive packets of interest on EndaceProbe.
-        Archived packets can later be downloaded from EndaceProbe as a PCAP file.
-        Archived Files never expire. Allowed chars are text, numbers, dash and underscore.
-      script: Endace|||endace-create-archive
-      type: regular
-      iscommand: true
-      brand: Endace
-    nexttasks:
-      '#none#':
-      - "9"
-    scriptarguments:
-      archive_filename:
-        simple: event
-      dest_host_list: {}
-      dest_port_list: {}
-      end: {}
-      protocol: {}
-      src_host_list:
-        simple: 10.16.13.188
-      src_port_list: {}
-      start: {}
-      timeframe:
-        simple: "3600"
-    separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 1580
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-view: |-
-  {
-    "linkLabelsPosition": {},
-    "paper": {
-      "dimensions": {
-        "height": 3615,
-        "width": 380,
-        "x": 50,
-        "y": 50
-      }
-    }
-  }
-inputs: []
-outputs: []
+id: Endace-Test
```

Comment:
cant really review this playbook
what changed here?

Reply:
I have no idea why it shows the diff like that, try the split view maybe

The only changes here are added description where necessary and changed id to -1